### PR TITLE
Fixes flaky script

### PR DESCRIPTION
**Pull request by chunhtai**
fixes https://github.com/flutter/flutter/issues/87500

script will now add team label
Skip filing the pr if no entry in ci.yaml
updated ignored list for deflake script

### CODE REVIEW

File: app_dart/lib/src/request_handlers/flaky_handler_utils.dart [hunk 1]
```diff
@@ -74,11 +85,16 @@ Please follow https://github.com/flutter/flutter/wiki/Reducing-Test-Flakiness#fi
...
 
   List<String> get issueLabels {
-    return <String>[
+    final List<String> labels = <String>[
       kTeamFlakeLabel,
       kSevereFlakeLabel,
       kP1Label,
     ];
+    final String teamLabel = _getTeamLabelFromTeam(ownership.team);
+    if (teamLabel?.isNotEmpty == true) {
```
Comment (keyonghan) [inline app_dart/lib/src/request_handlers/flaky_handler_utils.dart:94]: Nit: skip `== true`.

File: app_dart/lib/src/request_handlers/flaky_handler_utils.dart [hunk 2]
```diff
@@ -425,10 +454,56 @@ String _issueBuildLink({String builder, String build}) {
...
+      return kFrameworkLabel;
+    case Team.engine:
+      return kEngineLabel;
+    case Team.tool:
+      return kToolLabel;
+    case Team.web:
+      return kWebLabel;
+    case Team.unknown:
+      return null;
+  }
```
Comment (keyonghan) [inline app_dart/lib/src/request_handlers/flaky_handler_utils.dart:483]: A return is needed out of the switch block.

**Review by keyonghan — COMMENTED** [review]
Seems CI is failing.

File: app_dart/lib/src/request_handlers/flaky_handler_utils.dart (hunk 1 above)
Comment (chunhtai) [reply app_dart/lib/src/request_handlers/flaky_handler_utils.dart:94]: is
```dart
if (teamLabel != null && teamLabel!. isNotEmpty)
```
prefer over this?

In flutter/flutter we usually do this instead

Comment (keyonghan) [reply app_dart/lib/src/request_handlers/flaky_handler_utils.dart:94]: TIL. Let's keep it with flutter/flutter then.

File: app_dart/lib/src/request_handlers/flaky_handler_utils.dart (hunk 2 above)
Comment (chunhtai) [reply app_dart/lib/src/request_handlers/flaky_handler_utils.dart:483]: fixed

**Review by keyonghan — APPROVED** [review]
LGTM with tests pass.

**Pull request merged**